include/dbstructure.php

```diff
@@ -464,13 +464,13 @@ function db_definition($charset) {
 			);
 	$database["cache"] = array(
 			"fields" => array(
-					"k" => array("type" => "varchar(255)", "not null" => "1", "primary" => "1"),
+					"k" => array("type" => "varbinary(255)", "not null" => "1", "primary" => "1"),
 					"v" => array("type" => "text"),
 					"expire_mode" => array("type" => "int(11)", "not null" => "1", "default" => "0"),
 					"updated" => array("type" => "datetime", "not null" => "1", "default" => "0000-00-00 00:00:00"),
 					),
 			"indexes" => array(
-					"PRIMARY" => array("k".db_index_suffix($charset)),
```

Comment:
Yes, this used to limit the index size to 191, the maximum number of 4-bytes characters InnoDB supports for a UNIQUE index.